sdk/core/System.ServiceModel.Rest/src/Pipeline/HttpPipelineMessageTransport.cs

```diff
@@ -59,23 +59,11 @@ private static HttpClient CreateDefaultClient()
         };
     }
 
-    public override PipelineMessage CreateMessage(InvocationOptions options)
+    public override PipelineMessage CreateMessage()
     {
         PipelineRequest request = new HttpPipelineRequest();
         PipelineMessage message = new PipelineMessage(request);
 
-        // Wire up options on message
-        message.CancellationToken = options.CancellationToken;
-        message.ResponseClassifier = options.ResponseClassifier;
```

Comment:
This got me in a lot of trouble

Reply:
Temp fix with this: https://github.com/Azure/azure-sdk-for-net/pull/39192/commits/7662212dcce2e437b29989352e94bec8b3e48706#diff-80fbeb5f2ad62c5d4d4c4beaeb1aea6e30417002292e1fcab0319816f754c878